app/Http/Controllers/PokerController.php

```diff
@@ -1,133 +1,133 @@
-<?php
+    <?php
 
-namespace App\Http\Controllers;
+    namespace App\Http\Controllers;
 
-use Illuminate\Http\Request;
-use Illuminate\Support\Facades\DB;
-use Illuminate\Support\Facades\Broadcast;
-use Illuminate\Support\Facades\Log;
+    use Illuminate\Http\Request;
+    use Illuminate\Support\Facades\DB;
+    use Illuminate\Support\Facades\Broadcast;
+    use Illuminate\Support\Facades\Log;
 
-class PokerController extends Controller
-{
+    class PokerController extends Controller
+    {
 
-    public function createRoom(Request $request){
+        public function createRoom(Request $request){
 
-        $room_id = $this->generateRandomString(); //todo make sure this is unique in db        
+            $room_id = $this->generateRandomString(); //todo make sure this is unique in db        
 
-        $now = date('Y-m-d H:i:s');
-        $id = DB::table('poker_rooms')->insertGetId([
-            'room_id' => $room_id,
-            'password' => $request->password,
-            'is_open' => true,
-            'created_at' => $now,
-            'updated_at' => $now
-        ]);
-
-        $one_week_ago = date("Y-m-d H:i:s", strtotime("-1 week"));
+            $now = date('Y-m-d H:i:s');
+            $id = DB::table('poker_rooms')->insertGetId([
+                'room_id' => $room_id,
```

Comment:
which constants are you talking about specifically

Reply:
`'password'`, `'is_open'`, `'created_at'`, `'updated_at'`, `'room_id'`
Since these denote table columns but are also used as variable names, it could be cleaner to define them as static constants, ex:

`PASSWORD_COL`, `IS_OPEN_COL`, `CREATED_AT_COL`,